examples/run_models/configs/caduceus_config.yaml

```diff
@@ -0,0 +1,5 @@
+model_name: "caduceus-ph-4L-seqlen-1k-d118"
+batch_size: 2
+pooling_strategy: "mean"
+device: "cuda"
```

Comment:
If cuda GPU is a strict requirement, I would take it out here and hardcode it after the check in `Caduceaus.__init__()`. Otherwise the user might try `cpu`, which does not work